packages/app-mobile/utils/ShareUtils.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, test, expect } from '@jest/globals';
+import { mkdtempSync, rmdirSync } from 'fs';
```

Comment:
Please don't use Sync calls, especially in tests, since it could prevent them from running in parallel. The async calls should work fine with Jest

Reply:
I'm using `mkdtempSync` to create a directory before another import statement runs:
```
const tempDirectoryPath = mkdtempSync(join(tmpdir(), 'ShareUtilsTest'));
jest.mock('react-native-fs', () => {
	return {
		CachesDirectoryPath: tempDirectoryPath,
	};
});

// Imports that might use the react-native-fs mock
import { writeTextToCacheFile } from './ShareUtils';
```
Here are two other ways I could try to write this:
1. Delay the import using ES6 `import(...)` syntax.
2. Re-write `ShareUtils.ts` to not use `CachesDirectoryPath` directly. This could instead be a property exposed by `shim`.


Additionally, I'm using `existsSync` because `fs.exists` [is deprecated](https://nodejs.org/api/fs.html#fsexistspath-callback).

Reply:
Couldn't it be in beforeEach? In which case you can use async calls.

Regarding `exists`, usually I just use `fs-extra::pathExists` which is async and non-deprecated

Reply:
This has been done!